### DISCUSSION TITLE
Style for mid panel

### DISCUSSION THREAD

**Discussion by SimpleFoxOfficial** [BEEmod/BEE2-items#4137] [Feature Suggestions]
![image](https://user-images.githubusercontent.com/82114463/166413477-16ce06b5-1eaf-49dc-82b5-d002f5552fc7.png)
![image](https://user-images.githubusercontent.com/82114463/166413504-b9dab447-16ea-4d64-b937-468811d115d7.png)
You can see them in 8 chapter of portal 2 and in this beta footage. I think this would be cool like style for middle panel

**Comment by SimpleFoxOfficial** [comment]
![image](https://user-images.githubusercontent.com/82114463/166414185-77bfe203-69e8-4c8a-839e-73470097cd08.png)
also

**Reply by ItzAlmaZ** [reply]
I wanted to suggest it too. A panel with an aspect ratio of 3:1 or 2:1 (2&3 height, and 1 width)
